docs/git.md

```diff
@@ -54,3 +54,23 @@ You can do so by typing :
 echo ".idea" >> ~/.gitignore_global
 git config --global core.excludesfile ~/.gitignore_global
 ```
+
+## How to get the .git history of an installed vendor
+
+If you wish to work on a bundle inside a project, we advise getting his git sources directly inside the project, rather than cloning it and open in a standalone context.
+That way you can directly test your changes in a real project situation, commit and push these changes to the bundle repository remote.
+
+Firstly you must require the bundle you want to work on the "normal" way.
+```shell
+composer req smartbooster/symfony-docker
```

Comment:
The smartbooster/symfony-docker isn't something we install with composer. 
Instead, take the standard-bundle as a good example:

```shell
composer require --dev smartbooster/standard-bundle
```